coa_flask_app/__init__.py

```diff
@@ -156,4 +156,53 @@ def locations_hierarchy() -> JSON:
     Returns:
         A json list of the locations hierarchy.
     """
-    return jsonify(locationsHierarchy=coa_logic.locations_hierarchy())
+    return jsonify(locationsHierarchy=site.locations_hierarchy())
+
+
+@APP.route('/getTLs')
+def get_tls() -> JSON:
+    """
+    The get tls route returns the all the TLs for the input drop down.
```

Comment:
Minor:  Might be nice to change it to "The get tls route returns the all the team leads for the input drop down."